DQMServices/Core/python/DQMEDAnalyzer.py

```diff
@@ -1 +1 @@
-from FWCore.ParameterSet.Config import EDAnalyzer as DQMEDAnalyzer
+from FWCore.ParameterSet.Config import EDProducer as DQMEDAnalyzer
```

Comment:
I guess this change should be dropped.

Reply:
yes, that slipped in with the cherry-pick. Not sure how, I thought I had that in a separate commit...